aldryn_events/conf.py

```diff
@@ -14,6 +14,7 @@ class EventsAppConf(AppConf):
     MANAGER_REGISTRATION_EMAIL = False
     MANAGERS = None
     DEFAULT_FROM_EMAIL = None
+    PLUGIN_CACHE_TIMEOUT = 60
```

Comment:
At a minimum, this should probably be 900 (15 minutes), but we should consider 3600 (1 hour) here instead.